polyfill/lib/ecmascript.mjs

```diff
@@ -311,6 +312,14 @@ export const ES = ObjectAssign(ObjectAssign({}, ES2019), {
       'reject'
     );
   },
+  ToLimitedDuration: (item, disallowedSlots = []) => {
+    const duration = ES.ToDuration(item);
+    for (let slot of disallowedSlots) {
+      if (GetSlot(duration, slot) !== 0)
+        throw new RangeError(`invalid duration field ${slot}`);
```

Comment:
Nit: I think we consistently brace single-line ifs. (We should perhaps lint this code.)

Reply:
I think the current state is pretty inconsistent, how about I open a separate PR adding an eslintrc and the results of running it over the code?

Reply:
See #394 for the linter PR; I'll make this change and merge this PR now.